source/docs/articles/sites/migrate/migrating-from-wpengine.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Migrating WordPress Sites from WP Engine to Pantheon
+description: Get all the details you need to know to successfully migrate your site away from WP Engine.
+category:
+  - WordPress
+  - getting-started
+  - developing
+keywords: wordpress, pantheon, wpengine
+---
+
+Manually migrating a site from WP Engine to Pantheon can be tricky, as you cannot use WP Engine’s backup feature to transfer your site. This is because the backup file excludes many of your site's files and directories. See the full list of [exclusions](https://wpengine.com/support/restore/).
+
+The easiest way to migrate your site is to use the [Pantheon Migration Plugin](https://wordpress.org/plugins/bv-pantheon-migration/) created by BlogVault.
+
+## Migrate Using the Pantheon Migration Plugin
+
+1. You can install the plugin by uploading `bv-pantheon-migration` through the WordPress Plugins menu or to the `/wp-content/plugins/` directory.
```

Comment:
Does the plugin get installed to the WPEngine site or to the Pantheon site, or to both?